LICENSE

```diff
@@ -174,34 +174,6 @@
       incurred by, or claims asserted against, such Contributor by reason
       of your accepting any such warranty or additional liability.
 
-   END OF TERMS AND CONDITIONS
```

Comment:
We might need to add this line?

Reply:
I think it’s fine to add back in since there is text under the primary license. I’ll make that change.

Reply:
Updated. I'll let you resolve this conversation after you review.